{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2d215940",
   "metadata": {},
   "outputs": [
    {
     "ename": "",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31mThe kernel failed to start as the Python Environment 'text2seg (Python -1.-1.-1)' is no longer available. Consider selecting another kernel or refreshing the list of Python Environments."
     ]
    }
   ],
   "source": [
    "from pymongo import MongoClient\n",
    "\n",
    "# URL format: mongodb://<username>:<password>@<host>:<port>/\n",
    "client = MongoClient(\"mongodb://admin:secret123@localhost:27017/\")\n",
    "db = client[\"product_db\"]\n",
    "collection = db[\"products\"]\n",
    "\n",
    "# Test insert\n",
    "collection.insert_one({\"name\": \"red shoe\", \"path\": \"data/red_shoe.jpg\"})\n",
    "print(list(collection.find()))\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "text2seg",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "-1.-1.-1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
